{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0",
    "cveMetadata": {
        "cveId": "CVE-2023-38252",
        "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "state": "PUBLISHED",
        "assignerShortName": "redhat",
        "dateReserved": "2023-07-13T16:29:56.474Z",
        "datePublished": "2023-07-14T17:06:20.732Z",
        "dateUpdated": "2024-03-18T03:38:19.029Z"
    },
    "containers": {
        "cna": {
            "title": "W3m: out of bounds read in strnew_size() at w3m/str.c",
            "metrics": [
                {
                    "other": {
                        "content": {
                            "value": "Low",
                            "namespace": "https://access.redhat.com/security/updates/classification/"
                        },
                        "type": "Red Hat severity rating"
                    }
                },
                {
                    "cvssV3_1": {
                        "attackComplexity": "HIGH",
                        "attackVector": "LOCAL",
                        "availabilityImpact": "HIGH",
                        "baseScore": 4.7,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "NONE",
                        "scope": "UNCHANGED",
                        "userInteraction": "REQUIRED",
                        "vectorString": "CVSS:3.1/AV:L/AC:H/PR:N/UI:R/S:U/C:N/I:N/A:H",
                        "version": "3.1"
                    },
                    "format": "CVSS"
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "An out-of-bounds read flaw was found in w3m, in the Strnew_size function in Str.c. This issue may allow an attacker to cause a denial of service through a crafted HTML file."
                }
            ],
            "affected": [
                {
                    "product": "w3m",
                    "vendor": "n/a",
                    "defaultStatus": "affected"
                },
                {
                    "vendor": "Red Hat",
                    "product": "Red Hat Enterprise Linux 6",
                    "collectionURL": "https://access.redhat.com/downloads/content/package-browser/",
                    "packageName": "w3m",
                    "defaultStatus": "unknown",
                    "cpes": [
                        "cpe:/o:redhat:enterprise_linux:6"
                    ]
                },
                {
                    "product": "Fedora",
                    "vendor": "Fedora",
                    "collectionURL": "https://packages.fedoraproject.org/",
                    "packageName": "w3m",
                    "defaultStatus": "affected"
                },
                {
                    "product": "Extra Packages for Enterprise Linux",
                    "vendor": "Fedora",
                    "collectionURL": "https://packages.fedoraproject.org/",
                    "packageName": "w3m",
                    "defaultStatus": "affected"
                }
            ],
            "references": [
                {
                    "url": "https://access.redhat.com/security/cve/CVE-2023-38252",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_REDHAT"
                    ]
                },
                {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2222775",
                    "name": "RHBZ#2222775",
                    "tags": [
                        "issue-tracking",
                        "x_refsource_REDHAT"
                    ]
                },
                {
                    "url": "https://github.com/tats/w3m/issues/270"
                },
                {
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/AULOBQJLXE2KCT5UVQMKGEFL4GFIAOED/"
                }
            ],
            "datePublic": "2023-06-29T00:00:00+00:00",
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-125",
                            "description": "Out-of-bounds Read",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "x_redhatCweChain": "CWE-125: Out-of-bounds Read",
            "timeline": [
                {
                    "lang": "en",
                    "time": "2023-06-29T00:00:00+00:00",
                    "value": "Reported to Red Hat."
                },
                {
                    "lang": "en",
                    "time": "2023-06-29T00:00:00+00:00",
                    "value": "Made public."
                }
            ],
            "providerMetadata": {
                "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
                "shortName": "redhat",
                "dateUpdated": "2024-03-18T03:38:19.029Z"
            }
        }
    }
}